core/src/main/java/it/gov/pagopa/bpd/citizen/service/CitizenServiceImpl.java

```diff
@@ -57,17 +57,26 @@ public Citizen find(String fiscalCode) {
     public Citizen update(String fiscalCode, Citizen cz) {
```

Comment:
@daniloMessinaSia Of course you are right, but I think the _onboarding_ process won't be re-enabled. Anyway - just in case - I'll fix also the delete method